### DISCUSSION TITLE
Export Errors

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8681] [Help]
Hi there,

I'm new with BB, so I might be doing something wrong. I'v created an button with an Export action ...

![Screenshot 2022-11-14 at 07 58 14](https://user-images.githubusercontent.com/4005405/201595069-628d6047-2665-4064-913a-62d65220c816.png)

When I preview this page, I get the first error, being 'Please select at least one row'. I then select a few rows and click the export button, and then get the errors 'Unexpected token ' in JSON at position 1' and 'There was an error exporting the data'.

![Screenshot 2022-11-14 at 08 00 37](https://user-images.githubusercontent.com/4005405/201595416-ac081b3c-d6d6-47d7-9a17-c22223542058.png)

My data is coming from a PostgreSQL (/Supabase) database.

Can anybody please tell me what I might be doing wrong, and/or how to do this properly/fix it?

I'm currently reviewing BB, and I like what I see so far. But not being able to export data is a show stopper for me.

Thanks in advance and best regards, Dick

**Comment by melohagan** [comment]
Hey @ecxs 

What are the data types of *node* and *workflow* ?

**Reply by ecxs** [reply]
Hey @melohagan 

Both text

![Screenshot 2022-11-21 at 12 11 20](https://user-images.githubusercontent.com/4005405/203036136-dc6ed9df-ce63-4289-8959-bbae469219f5.png)

**Reply by melohagan** [reply]
What version of Budibase is your app on?

**Reply by ecxs** [reply]
2.1.28

**Reply by melohagan** [reply]
I wasn't able to recreate this locally. Would you be able to share an app export that reproduces the problem please?

**Reply by ecxs** [reply]
I'm happy to share a copy with you, but since I'm new with Budibase, I was wondering whether the export holds confidential information like credentials to the external data sources?

**Reply by melohagan** [reply]
Hey @ecxs It would. If you're able to setup a local version of your app with some dummy data against a local database, it would be helpful for me to be able to replicate the problem locally as well.

**Reply by ecxs** [reply]
Hey @melohagan is there a way to easily replicate a PostgreSQL table to an local table?

**Reply by melohagan** [reply]
If you're using pgadmin, you should be able to right click the table to get the CREATE script, which you could then run in a local instance. Then it's just a matter of inserting some dummy data.

<img width="421" alt="Screenshot 2022-11-21 at 14 00 45" src="https://user-images.githubusercontent.com/101575380/203073827-3df8c543-8e80-47a1-95c4-73925f4f0168.png">

**Reply by ecxs** [reply]
@melohagan thanks for your instructions. I forgot to mention that I'm not actually using PostgreSQL in this example, but Supabase ...
